music/apiclientng.go

```
@@ -0,0 +1,274 @@
+	tdns "github.com/johanix/tdns/tdns"
+)
+
+func ExtractHoldPeriod(buf []byte) int {
+	var de DesecError
+	err := json.Unmarshal(buf, &de)
+	if err != nil {
+		log.Fatalf("Error from unmarshal DesecError: %v\n", err)
+	}
+	// "Request was throttled. Expected available in 1 second."
+	fmt.Printf("deSEC error detail: '%s'\n", de.Detail)
+	de.Detail = strings.TrimLeft(de.Detail, "Request was throttled. Expected available in ")
```

golangci-lint (1.62.2)

SA1024: cutset contains duplicate characters

(staticcheck)